METRICS.md

```diff
@@ -69,3 +69,8 @@ scrape_configs:
 - `hopr_chain_actions_count`: Number of different chain actions and their results, keys: `action`, `result`
 - `hopr_indexer_contract_log_counters`: Counts of different HOPR contract logs processed by the Indexer, keys: `contract`
 - `hopr_tickets_incoming_statistics`: Ticket statistics for channels with incoming tickets, keys: `channel`, `statistic`
+- `hopr_session_num_active_session`: Number of currently active HOPR sessions
```

Comment:
Should we consider splitting these stats up into tcp and udp sessions?

Reply:
At this level it's only data. It becomes TCP/UDP only at entry or exit.

Reply:
Added additional metrics to hoprd.